src/genesys/verifiers/swe_fixer_verfier.py

```diff
@@ -0,0 +1,192 @@
+### adapted from https://github.com/InternLM/SWE-Fixer/blob/main/evaluation/code_edit.py
+import json
+import os
+import re
+import ast
+import argparse
+
+from genesys.schemas import Response
+from genesys.verifiers.base_verifier import BaseVerifier
+
+
+def parse_json_codeblock_from_reasoning_model_output(markdown_str):
+    _, answer_str = markdown_str.split("</think>")
```

Comment:
This will fail to unpack and throw an exception if there's no </think>, which I don't think there's anything to process atm.